diff --git a/types/index.d.ts b/types/index.d.ts
index 26edbb2575a5e2c8d6149c6935068b3be6012cc9..a2f4ad08d572bec2bf4ca34ba6352d240ee00bb4 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -444,7 +444,7 @@ export declare namespace knex {
   class QueryBuilder {
     static extend(
       methodName: string,
-      fn: <TRecord extends {} = any, TResult = unknown[]>(
+      fn: <TRecord extends {} = any, TResult extends {} = unknown[]>(
         this: Knex.QueryBuilder<TRecord, TResult>,
         ...args: any[]
       ) =>
