diff --git a/node_modules/vue-loader/dist/index.d.ts b/node_modules/vue-loader/dist/index.d.ts
index 16d88dd..2d0e357 100644
--- a/node_modules/vue-loader/dist/index.d.ts
+++ b/node_modules/vue-loader/dist/index.d.ts
@@ -13,4 +13,4 @@ export interface VueLoaderOptions {
     appendExtension?: boolean;
     isServerBuild?: boolean;
 }
-export default function loader(this: webpack.loader.LoaderContext, source: string): string | void;
+export default function loader(this: webpack.LoaderContext<string>, source: string): string | void;
diff --git a/node_modules/vue-loader/dist/plugin.d.ts b/node_modules/vue-loader/dist/plugin.d.ts
index 6e03582..dcc32d4 100644
--- a/node_modules/vue-loader/dist/plugin.d.ts
+++ b/node_modules/vue-loader/dist/plugin.d.ts
@@ -1,5 +1,5 @@
 import webpack = require('webpack');
-declare class VueLoaderPlugin implements webpack.Plugin {
+declare class VueLoaderPlugin implements webpack.WebpackPluginInstance {
     static NS: string;
     apply(compiler: webpack.Compiler): void;
 }
